config/routes.rb

```diff
@@ -1,20 +1,31 @@
 Rails.application.routes.draw do
+  get 'stores/new'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.